source/layouts/layout.erb

```diff
@@ -37,52 +46,105 @@ under the License.
   </head>
 
   <body class="<%= page_classes %>" data-languages="<%=h language_tabs.map{ |lang| lang.is_a?(Hash) ? lang.keys.first : lang }.to_json %>">
-    <a href="#" id="nav-button">
-      <span>
-        NAV
-        <%= image_tag('navbar.png') %>
-      </span>
-    </a>
-    <div class="tocify-wrapper">
+    
+    <div class="ods__documentation-header">
+    
+      <div class="ods__documentation-header-logo">
+        <button type="button" id="nav-button"> Menu </button>
+        <a href="">
+          <img src="../images/ODS_logo api blanc.svg" alt="ods-documentation-logo">
+        </a>
+        <button type="button" id="help-hub-button"> Help Hub </button>
+      </div>
+
+      <div class="ods__documentation-header-nav">
+        <div class="ods__documentation-header-nav-item">
+          <a href="http://docs.opendatasoft.com/en/">Platform</a>
+        </div>
+        <div class="ods__documentation-header-nav-item">
+          <a href="https://discovery.opendatasoft.com/pages/home/">Discovery</a>
+        </div>
+        <div class="ods__documentation-header-nav-item">
+          <a href="https://docs.opendatasoft.com/en/faq.html">FAQ</a>
+        </div>
+        <div class="ods__documentation-header-nav-item">
+          <a href="https://opendatasoft.github.io/ods-widgets/docs/#/api">Widgets</a>
+        </div>
+        <div class="ods__documentation-header-nav-item">
+          <a href="https://opendatasoft.github.io/ods-widgets/docs/#/tutorial">Widgets Tutorial</a>
+        </div>
+        <div class="ods__documentation-header-nav-item ods__documentation-header-nav-item-active">
+          <a href="https://docs.opendatasoft.com/api/explore/v2.html#search-api-v2">API</a>
+        </div>
+      </div>
+    
+    </div>
+
+    <div class="ods__documentation-help-hub-sidebar">
+      <div class="ods__documentation-help-hub-sidebar-item">
+          <a href="https://discovery.opendatasoft.com/pages/home/">Platform</a>
+        </div>
+        <div class="ods__documentation-help-hub-sidebar-item">
+          <a href="https://discovery.opendatasoft.com/pages/home/">Discovery</a>
+        </div>
+        <div class="ods__documentation-help-hub-sidebar-item">
+          <a href="https://docs.opendatasoft.com/en/faq.html">FAQ</a>
+        </div>
+        <div class="ods__documentation-help-hub-sidebar-item">
+          <a href="https://opendatasoft.github.io/ods-widgets/docs/#/api">Widgets</a>
+        </div>
+        <div class="ods__documentation-help-hub-sidebar-item">
+          <a href="https://opendatasoft.github.io/ods-widgets/docs/#/tutorial">Widgets Tutorial</a>
+        </div>
+        <div class="ods__documentation-help-hub-sidebar-item ods__documentation-header-nav-item-active">
+          <a href="https://docs.opendatasoft.com/api/explore/v2.html#search-api-v2">API</a>
+        </div>
+    </div>
```

Comment:
This is a pure code duplicate, do you think we could instead use media queries to change the layout?